Library/Formula/input-source-switcher.rb

```diff
@@ -0,0 +1,20 @@
+class InputSourceSwitcher < Formula
+  homepage "https://github.com/vovkasm/input-source-switcher"
+  url "https://github.com/vovkasm/input-source-switcher/archive/cfca8149c6805b333910858a22aeeb124cbca903.zip"
```

Comment:
Is there a stable release for this software? We only accept software with stable releases.